explorer/templates/explorer/query_favorite_button.html

```diff
@@ -0,0 +1,8 @@
+{% if is_favorite %}
+    <i class="glyphicon glyphicon-heart text-danger query_favorite_toggle" data-id="{{ query_id }}"
```

Comment:
With the template tag passing in extra classes; 
```suggestion
    <i class="glyphicon glyphicon-heart text-danger {{ extra_classes }}" data-id="{{ query_id }}"
```